src/Roulette/Main.java

```diff
@@ -0,0 +1,113 @@
+                    bet = type.nextInt();
+                    if (bet > myAccount.getAccountBalance() || bet<0) {
+                        do {
+                            System.out.println("Enter the correct amount of bet");
+                            bet = type.nextInt();
+                        } while (bet > myAccount.accountBalance() || bet<0);
+                    }
+                    myAccount.setAccountBalance(myAccount.getAccountBalance()-bet);
+                    System.out.println("Choose the color you want to bet on: green, red or black");
+                    colorBet = type.next();
+                    if(!colorBet.equals("green") && !colorBet.equals("red") && !colorBet.equals("black")){
+                        do{
+                            System.out.println("Choose the correct color: green, red or black");
+                            colorBet = type.next();
+                        }while(!colorBet.equals("green") && !colorBet.equals("red") && !colorBet.equals("black"));
+                    }
+                    drawnNumber = rouletteStart();
+                    if (drawnNumber == 0) {
+                        color = "green";
+                    } else if (drawnNumber % 2 == 1) {
+                        color = "red";
+                    } else {
+                        color = "black";
+                    }
+                    if (drawnNumber == 0 && colorBet.equals("green")){
+                        bet = bet * 36;
+                        System.out.println("You won " + bet + " dollars");
+                        myAccount.setAccountBalance(myAccount.getAccountBalance() + bet);
+                        bet = 0;
+                    } else if (drawnNumber == 0) {
+                        bet = 0;
+                        System.out.println("You lost, the color was " + color);
+                    } else if (drawnNumber % 2 == 1 && colorBet.equals("red")) {
+                        bet = bet * 2;
+                        System.out.println("You won " + bet + " dollars");
+                        myAccount.setAccountBalance(myAccount.getAccountBalance() + bet);
+                        bet = 0;
+                    } else if (drawnNumber % 2 == 1 && !colorBet.equals("red")) {
+                        bet = 0;
+                        System.out.println("You lost, the color was " + color);
+                    } else if (drawnNumber % 2 == 0 && colorBet.equals("black")) {
+                        bet = bet * 2;
+                        System.out.println("You won " + bet + " dollars");
+                        myAccount.setAccountBalance(myAccount.getAccountBalance() + bet);
+                        bet = 0;
+                    } else if (drawnNumber % 2 == 0 && !colorBet.equals("black")) {
+                        bet = 0;
+                        System.out.println("You lost, the color was " + color);
+                    }
+                    break;
```

Comment:
private static void processWonBet(int bet, int multipleRatio, Roulette.Account myAccount) {
        bet = bet * multipleRatio;
        System.out.println("You won " + bet + " dollars");
        myAccount.setAccountBalance(myAccount.getAccountBalance() + bet);
    }